### DISCUSSION TITLE
Complex filters/queries combining tags, categories and accounts?

### DISCUSSION THREAD

**Discussion by Ezwen** [firefly-iii/firefly-iii#6757] [Firefly III]
Let's say I have:
- Two revenue accounts: R1 and R2
- Two categories: C1 and C2
- Two tags: T1 and T2

Is there any way I can get the list of operations (and the sum) for a given combination?

For instance, I would like to get "all operations categorized C1, tagged T2, coming from both R1 and R2"

**Comment by JC5** [comment]
You can make account and category or tag filtered reports under `/reports`, and the search is pretty expansive (`/search`), but there's not really a "any given combination" kind of thing.

**Reply by Ezwen** [reply]
I see thanks! The search part is pretty great, is there any way to get the sum of all amounts of all operations retrieved by a search request?

**Reply by JC5** [reply]
No I'm afraid not, but I'm working on it!
